src/BigBlueButton.php

```diff
@@ -108,17 +113,19 @@ class BigBlueButton
      *
      * @throws ConfigException
      */
-    public function __construct(?string $baseUrl = null, ?string $secret = null, ?TransportInterface $transport = null)
+    public function __construct(?string $baseUrl = null, ?string $secret = null, ?TransportInterface $transport = null, ?string $hashAlgorithm = 'sha1')
```

Comment:
```suggestion
    public function __construct(?string $baseUrl = null, ?string $secret = null, ?TransportInterface $transport = null, string $hashAlgorithm = 'sha1')
```

It does not make sense to allow `null` here.